src/components/erc721/marketplace/trade_button.tsx

```diff
@@ -35,16 +35,16 @@ BtnStyled.defaultProps = {
 interface Props {
     asset: Collectible;
     ethAccount: string;
-    isDisabled: boolean;
+    isDisabled?: boolean;
```

Comment:
Q: Why was this made optional?

Reply:
> Q: Why was this made optional?

It is to avoid a linter problem, later use a spread operator in the component.